mlc_config.json

```diff
@@ -14,6 +14,9 @@
         },
         {
             "pattern": "^https://source.android.com/setup/develop/repo$"
+        },
+        {
+            "pattern": "^../../tutorial/intro$"
```

Comment:
Please explain in the commit message *why* this needs to be excluded, esp. as this is a local / relative link.